src/baseConn.ts

```diff
@@ -104,6 +111,9 @@ export class BaseRawConnection {
   protocolVersion: ProtocolVersion = PROTO_VER;
   isLegacyProtocol = false;
 
+  protected stateCodec: ICodec = INVALID_CODEC;
```

Comment:
Looks like `INVALID_CODEC` is now only used as an initial value, and the `INVALID_CODEC_ID` is never used. Maybe we can avoid calling it "invalid" as we don't have (or - at least - don't recommend) to use the `FFFF...` type id hack.

Reply:
Ah right. I remember @elprans mentioned in one of the meetings that he's got some idea to handle the state desc for binary-over-http connections, I suppose this is it?